sass/_header.scss

```diff
@@ -207,3 +187,50 @@
         }
     }
 }
+
+.submenu-checkbox {
+    display: none;
+
+    @media screen and (max-width: 1023px) {
+        &:not(:checked) + .submenu-title-container + .section-submenu-wrap {
+            display: none;
+        }
+
+        &:not(:checked) + .submenu-title-container {
+            .arrow {
+                background-image: url("/assets/down-arrow.svg");
+                width: 1rem;
+                height: 1rem;
+                margin: auto 0;
+                filter: invert(1);
```

Comment:
Are we using the same image as black colored elsewhere? I'm almot inclined to recomment `{% include %}`ing it instead of using `background-image` and setting `fill: #fff` via CSS on the contained `<path>`. Also why does that file contain more than a dozen empty SVG groups? o_O

Reply:
We are using the same image black coloured in the documentation yes. Is it more efficient to use `fill` than `background-image` and `invert`?

I'm going to give [scour](https://github.com/scour-project/scour) a go on these assets :)

Reply:
```
Scour processed file "down-arrow.svg" in 12 ms: 602/857 bytes new/orig -> 70.2%
…
Scour processed file "up-arrow.svg" in 6 ms: 631/896 bytes new/orig -> 70.4%
```

Reply:
I'm not sure I parsed the last sentence correctly, I read it as "you're using the wrong tool for the job, it's not a problem" :D

Reply:
Yes, that is what I was saying. You are using a much more powerful tool than necessary to do the coloring, so there is more potential for issues, but in practice it will likely be perfectly fine.